src/components/navigation/Navigation.js

```diff
@@ -0,0 +1,126 @@
+/**
+ *
+ * @author Sriram
+ * A component that displays app level navigation
+ */
+import React from "react";
+import intlData from "./messages";
+import ids from "./ids";
+import { build, getMessage, withI18N } from "@cyverse-de/ui-lib";
+import CloudUploadIcon from "@material-ui/icons/CloudUpload";
+import AppsIcon from "@material-ui/icons/Apps";
+import DashboardIcon from "@material-ui/icons/Dashboard";
+import AssessmentIcon from "@material-ui/icons/Assessment";
+import MoreHorizIcon from "@material-ui/icons/MoreHoriz";
+import { Button, Grid, Hidden, Paper } from "@material-ui/core";
+import { useRouter } from "next/router";
+import { makeStyles, useTheme } from "@material-ui/core/styles";
+import { injectIntl } from "react-intl";
+
+const useStyles = makeStyles((theme) => ({
+    button: {
+        margin: 1,
+        "&:hover": {
+            backgroundColor: theme.palette.blue,
+            color: theme.palette.white,
+        },
+    },
+
+    selectedButton: {
+        margin: 1,
+        backgroundColor: theme.palette.blue,
+        color: theme.palette.white,
+    },
+}));
+
+const DASHBOARD = "dashboard";
+const DATA = "data";
+const APPS = "apps";
+const ANALYSES = "analyses";
+const MORE = "more";
```

Comment:
Since the other pages that use this component also have to reference these strings, maybe we should move this to a separate file that this component and the pages reference?